app/views/static_pages/dashboard_steps/community_rules.md

```diff
@@ -7,45 +7,58 @@ Our moderation team is committed to providing a friendly, safe and welcoming env
 We expect everyone to handle themselves in a professional manner and follow the spirit of our community rules. If you are given a warning by a member of our moderation team, please do not take it personally or be argumentative. We truly want to help you learn how to interact with others professionally, but your second chance does not have priority over the well-being of our community.
 
 #### 1. Please leave moderation to the moderators
+
 Please DM the @ModMail user with issues that need to be resolved with things on the server, like someone being a troll, or breaking rules in a way that requires intervention. Don't be afraid to let us know if something isn't right.
 
 #### 2. This community only supports the content in our curriculum
-All questions and advice should stay within the scope of our curriculum because we are only equipped to support our own material and recommendations. If you seek support on homework, personal projects, relationships, depression, mental illness, or anything else outside the scope of our curriculum, please seek out more equipped resources.
+
+All questions and advice should stay within the scope of our curriculum because we are only equipped to support our own material and recommendations. If you seek support on homework or personal projects, you should ask your teacher, a classmate, or another resource for help, even if the help you seek is for something covered by our curriculum.
```

Comment:
Good insight, but I think the wording is a bit hard to follow. What about something like this:
```suggestion
All questions and advice should stay within the scope of our curriculum because we are only equipped to support our own material and recommendations. Therefore, we do not support homework or personal projects, even if the topic is covered by our curriculum. You should ask your teacher, a classmate, or another resource for help.
```